sodetlib/operations/bias_steps.py

```diff
@@ -1170,7 +1170,7 @@ def take_bias_steps(S, cfg, bgs=None, step_voltage=0.05, step_duration=0.05,
 
         bsa.sid = sdl.stream_g3_on(
             S, tag=g3_tag, channel_mask=channel_mask, downsample_factor=1,
-            filter_disable=True, subtype=stream_subtype
+            filter_disable=True, subtype=stream_subtype, enable_compression=False
```

Comment:
Should `enable_compression` be a kwarg in take_bias_steps that's passed along to stream_g3_on instead of just always being set to False here?

Reply:
does it cause the data on disk to be much bigger such that always running these with compression off could become an issue in labs?

Reply:
Checking on the server, I believe the uncompressed bias step data is ~700 MB, which is large enough that we should be conscious of it in labs... I'll add the option to enable compression but I'm going to leave the default as False.

Reply:
done